Developers-Notes/conp-dataset_crawlers.md

```diff
@@ -0,0 +1,127 @@
+# conp-dataset crawlers
+
+The conp-dataset crawler scripts are located under the `scripts` directory of the [conp-dataset](https://github.com/CONP-PCNO/conp-dataset) repository.
+
+## Available crawlers
+
+There are crawlers available for the following platforms:
+
+- Zenodo (https://zenodo.org)
+- OSF (https://osf.io/)
```

Comment:
I thought we had crawlers for FRDR but it was not merged and Giulia is gone... Her PR is failing Circle CI and has conflict with the master branch... https://github.com/CONP-PCNO/conp-dataset/pull/401

Reply:
I guess this will bring another point: how do we upgrade the crawlers in the VM? @glatard ?

Reply:
I have a feeling I might be that someone once I figure it all out ;).